content-repo/extra-docs/articles/palo-alto-networks-cortex-xdr---investigation-and-response.md

```diff
@@ -0,0 +1,374 @@
+ 
+
+
+## In This Pack
+The Palo Alto Networks Cortex XDR - Investigation and Response content pack includes several content items.
+
+### Automations
+- **EntryWidgetNumberHostsXDR**: Entry widget that returns the number of hosts in a Cortex XDR incident. 
+- **EntryWidgetNumberUsersXDR**: Entry widget that returns the number of users that participated in a specified Cortex XDR incident.
+- **EntryWidgetPieAlertsXDR**: Entry widget that returns a pie chart of alerts for a specified Cortex XDR incident by alert severity (low, medium, and high).
+- **XDRSyncScript**: Deprecated. The incoming and outgoing mirroring feature added in XSOAR version 6.0.0 is used instead to sync XDR. After the Calculate Severity - Generic v2 sub-playbook’s run, Cortex XSOAR will be treated as the single source of truth for the severity field, and it will sync only from Cortex XSOAR to XDR, so manual changes for the severity field in XDR will not update in the XSOAR incident.
+
+### Classifiers
+- **Cortex XDR - Classifier**: Classifies Cortex XDR incidents.
+- **Cortex XDR - Incoming Mapper**: Maps incoming Cortex XDR incidents fields.
+- **Cortex XDR - Outgoing Mapper**: Maps outgoing Cortex XDR incidents fields.
+
+### Incident Types
+- **Cortex XDR Device Control Violations**
+- **Cortex XDR Disconnected endpoints**
+- **Cortex XDR Incident**
+- **Cortex XDR Port Scan**
+
+### Incident Fields
+- **LastMirroredInTime**
+- **XDR Alert Count**
+- **XDR Alerts**
+- **XDR Assigned User Email**
+- **XDR Assigned User Pretty Name**
+- **XDR Description**
+- **XDR Detection Time**
+- **XDR device control violations**
+- **XDR Disconnected endpoints**
+- **XDR File Artifacts**
+- **XDR High Severity Alert Count**
+- **XDR Host Count**
+- **XDR Incident ID**
+- **XDR Low Severity Alert Count**
+- **XDR manual severity**
+- **XDR Medium Severity Alert Count**
+- **XDR Modification Time**
+- **XDR Network Artifacts**
+- **XDR Notes**
+- **XDR Resolve Comment**
+- **XDR Status (Deprecated from version 6.0.0. Use XDR Status v2 instead)**
+- **XDR Status v2**
+- **XDR URL**
+- **XDR User Count**
+
+### Indicator Fields
+XDR status: The indicator status in XDR.
+
+### Integrations
+#### Cortex XDR - IOC
+Syncs indicators between Cortex XSOAR and Cortex XDR. 
+
+#### Palo Alto Networks Cortex XDR - Investigation and Response
+Enables direct execution of Cortex XDR actions within Cortex XSOAR.
+
+### Layouts
+There are 4 layouts in this pack. The information displayed in the layouts are similar with minor changes as detailed below.
+
+![XDR Case Info Tab](../../../docs/doc_imgs/reference/XDRLayout.png)
```

Comment:
@ShirleyDenkberg you are missing this image. That is why your build is failing. Please make sure to add the image.